### ISSUE TITLE
References per article, no misclassifiction, estimation of books

### ISSUE THREAD

**Issue by zuphilip** [locdb/locdb-journal-analysis#1]
Copy comments by @anlausch from the https://gist.github.com/zuphilip/e881f1bcbdabe7537eead67b0df72109 to discuss this further

---

- I added for each calculation a min/ max estimation. 
In the loop:
`number_of_references_per_article.append(nref)`
and then later:
```
print("Max", max(number_of_references_per_article))
print("Min", min(number_of_references_per_article))
```
- The has-references is not a misclassification; as I understand from the documentation, it just means that they are not openly accessible via the api. But you are right, for our purposes it is interesting, no matter how we name it.
- The estimation for books is a bit tricky. Why do we need it per page (the journal references are also not calculated per page)? With no identifiers, one could filter for the type of the resource and the year:
```py
works = Works(etiquette=my_etiquette).filter(has_references="true").filter(from_pub_date=2011).filter(until_pub_date=2011).filter(type="monograph")
for work in works:
    nref = work['reference-count']
    k += 1
    sum += nref
    number_of_references_per_resource.append(nref)
print("Max", max(number_of_references_per_resource))
print("Min", min(number_of_references_per_resource))
print("Number of monographs returned: ", k)
print("Sum of references: ", sum)
print("Average: ", sum/k)
```
We could do this for all crossref resource types, which you refer to by saying "book", i.e. book, monograph (more?). The same can be done for book chapters, of course.
The problem is that this is not related to the UniMA purchases nor domain-specific. Maybe category-name could help for the latter.

**Comment by zuphilip** [comment]
* The addition about the number of references per article sounds a nice addition and also a diagram about it (but we can do this later as well). @anlausch Do you want to push or make a PR of what you already have here?
* Should we replace the misclassification with closedReferences or something like this?
* I think that some similar calculations for books relating to our set is out-of-scope at the moment. But we have accurate number of the pages for the list of references there. Thus, I asked about the average number of references on a page. I looked more example through and still think that 20 is a good estimation for that.

**Comment by zuphilip** [comment]
2e4b45196bfe8fc684263230d6914d90e52bdcd9 solved the second point.

**Comment by zuphilip** [comment]
Closed by 341f08124d6ba0ae5cb53efe77327b1abf79e95b, 07e4383a3308b67da4471ab8af9db6364b47030c, 0aaa583bd510e478bd3c786b5bbeb3cf60213010. Thank you @anlausch !
